examples/chart/teleport-kube-agent/README.md

```diff
@@ -91,6 +91,26 @@ detailed below.
 
 ## Application access
 
+### Dynamic Registration mode
+
+To use Teleport application access in [dynamic registration mode](https://goteleport.com/docs/application-access/guides/dynamic-registration/),
+you will need to know the application resource selector. (`$APP_RESOURCE_KEY` and `$APP_RESOURCE_VALUE`)
+
+For example, to listen for all application resources set both variables to `*`.
+
+To install the agent in dynamic application registration mode, run:
+```sh
+$ helm install teleport-kube-agent . \
+--create-namespace \
+--namespace teleport \
+--set roles=app \
+--set proxyAddr=${PROXY_ENDPOINT?} \
+--set authToken=${JOIN_TOKEN?} \
+--set "appResources[0].labels.${APP_RESOURCE_KEY}=${APP_RESOURCE_VALUE}"
```

Comment:
```suggestion
--set "appResources[0].labels.${APP_RESOURCE_KEY?}=${APP_RESOURCE_VALUE?}"
```

Adding the `?` means that copy/pasting the command without providing those values will fail (which is desirable)